Vagrantfile

```diff
@@ -93,8 +94,10 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
     foreman.vm.network "private_network", ip: "172.28.128.20"
 
+    foreman.vm.synced_folder "custom-control-repo", "/etc/puppetlabs/facter/facts.d"
```

Comment:
Maybe change the source folder to `facts.d`. Would you also maybe be game to put a sample file in the directory that has a commented out entry for the custom control repo? Lastly, please add something to README.md about this.